translations/strings.yaml

```diff
@@ -600,6 +600,10 @@ tipsAndTricks:
   advancedTipsDescription: Let us help you learn more about Mozilla VPN
   quickTipsGuideViewTitle: Quick tip
   introModalDescription: Here’s a great list of useful tips and tricks to get the best out of your Mozilla VPN experience. Check it out now or find it anytime in the settings.
+  # Title for Tips & Tricks tabs
```

Comment:
This is not how comments are added to strings (check line 590).

Also, let's do "Tab title for one of the Tips & Tricks panels"